### DISCUSSION TITLE
HTML body final structure

### DISCUSSION THREAD

**Discussion by car2t** [commIT-Group/advice-generator-app#25] [Ideas]
Hi all!

I think we should finish the body structure before keep going with CSS or JS. We can always add more stuff later on in the `head` element and make some small class and IDs names modifications without much problem.

Every change in the document will affect the logic and styles so It will make our work unnecessarily difficult. I also think will make us waste more time.

What do you think? ;D

**Comment by caesiumtea** [comment]
What changes do you think still need to be made to the HTML? I thought that we already had all the HTML elements set up now, and all that's left is to use the CSS to arrange them.

**Reply by timiphil** [reply]
I don't know yet, I can check it out

**Reply by car2t** [reply]
It looked that it wasn't finish it yet, but not sure now.

The `script` element to the bottom of the body, that for sure, I have sent a commit and Phil did the changes too. Kung-fu fighting now xD

`<title>Advice generator app</title>` 

> Only the name of the app, too large and we already mention frontend mentor in the footer.

```
<footer>
      <p class="footer-text">
        Challenge by
        <a href="https://www.frontendmentor.io?ref=challenge" target="_blank"
          >Frontend Mentor</a
        >.
      </p>
      <p class="footer-text">
        Coded by <a href="https://github.com/commIT-Group">commIT Group</a>.
      </p>
</footer>
```
> This one could be (not dots for sure, no in headings and similar. Design thing)

```
<footer>
      <p class="footer-text">
        Challenge by<a href="https://www.frontendmentor.io?ref=challenge" target="_blank"
          >Frontend Mentor</a>
      </p>
      <p class="footer-text">Coded by</p>
      <address><a href="https://github.com/commIT-Group">commIT Group</a></address>
      
</footer>
```

> We could also wrap all this inside a `header`, but not sure if I would do it that way; no need for a `div`.

```
<h1>Advice #<span><---advicetext---></span></h1> //
<p id="advice-text">"<!-- Advice text goes here -->"</p>
<picture>...</picture>
```
> No need to send the `#` from JS. I even will put that sign from CSS using a pseudo element `span::before`, for example, or using its class name. Easier to maintain and we will keep the logic apart of the looks and structure.

But those maybe not really important. **The only thing I wanted to be sure is that the our HTML body structure is done.**

I hope make sense. 
Happy coding!!

**Reply by caesiumtea** [reply]
Oh, these are great suggestions, thanks! Good call with the address element, I'm always forgetting about semantics like that.

Adding the address element shouldn't conflict with anything, so somebody can insert that anytime. The span has to be coordinated with whoever is working on the Javascript though, right?

**Reply by car2t** [reply]
Yeah @caesiumtea Vance

After set the final HTML we should check the logic again, but not big deal.
Just say it and that's it! ;D
